src/main/java/edu/kis/powp/jobs2d/command/ComplexCommandFactory.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class ComplexCommandFactory {
+	public static ComplexCommand rectangleDrawing(DriverManager drivermanager) {
```

Comment:
DriverCommand return type would be sufficient

Passing a manager instead of a driver increases dependencies.